app/upgrades.go

```diff
@@ -9,6 +9,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	authztypes "github.com/cosmos/cosmos-sdk/x/authz/types"
```

Comment:
need this [key](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/x/authz/keys.go#L5) right?

Reply:
yes